### PR TITLE
DOC-638 Update uncommitted content

### PR DESCRIPTION
**Pull request by sfc-gh-mheavin**
Added content I accidentally forgot to push before merging mheavin-doc-636-php-windows-install

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -110,7 +116,7 @@ To build the PHP driver for Windows:
        git clone https://github.com/snowflakedb/pdo_snowflake.git
        cd pdo_snowflake
 
-#. Run the script to download the PHP SDK:
+#. Run the script to download the PHP source code:
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:119]: This step should be unchanged as it is downloading PHP SDK

Comment (sfc-gh-ext-simba-hx) [reply README.rst:119]: But the next step: `Download and install the PHP binaries, or build PHP yourself.` should be changed though, like 'Download and build PHP source code'

Comment (sfc-gh-mheavin) [reply README.rst:119]: Done

File: README.rst [hunk 2]
```diff
@@ -130,7 +136,7 @@ To build the PHP driver for Windows:
 
        .\\scripts\\setup_php_sdk.bat x64 Release VS16 C:\\php-sdk
 
-#. Download and install the PHP binaries, or build PHP yourself.
+#. Download and install the PHP binaries, or build the PHP source code.
 
    If you want to build PHP yourself, run the script to download the PHP source and build PHP:
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:141]: This one needs to be changed. There is no option to build or not build PHP, it's always needed. So maybe simply say "Run the script to download the PHP source and build PHP"

Comment (sfc-gh-mheavin) [reply README.rst]: Done

File: README.rst [hunk 3]
```diff
@@ -130,9 +136,9 @@ To build the PHP driver for Windows:
 
        .\\scripts\\setup_php_sdk.bat x64 Release VS16 C:\\php-sdk
 
-#. Download and install the PHP binaries, or build PHP yourself.
+#. Download and install the PHP binaries, or build the PHP source code.
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:139]: This one still not accurate. There is no option for customer to choose build or not build the PHP souce code. It's always needed as the driver build needs some artifacts from there. This step is not downloading and install PHP binaries. It's download and build PHP source code.

Comment (sfc-gh-mheavin) [reply README.rst]: Done

**Review by sfc-gh-ext-simba-hx — APPROVED** [review]
LGTM

File: README.rst [hunk 4]
```diff
@@ -97,9 +97,15 @@ You must install Microsoft Visual Studio 2019 (VS16) or earlier with the C++ dev
...
+#. Download and install PHP:
 
-    #. Download the PHP version binaries from `<https://windows.php.net/downloads/releases/>`_, such as `<https://windows.php.net/downloads/releases/php-8.1.18-Win32-vs16-x64.zip>`_.
+    #. Download the PHP version binaries from `<https://windows.php.net/downloads/releases/>`_, 
+       such as `<https://windows.php.net/downloads/releases/php-8.1.18-Win32-vs16-x64.zip>`_.
+
+       .. note::
+
+        The Snowflake PHP driver does not support Windows NTS, so don't download packages that
+        include *nts* in the package name.
```
Comment (sfc-gh-dyoshinaga) [inline README.rst:108]: Is nts a placeholder? If not, use:

```
``nts``
```

**Review by sfc-gh-dyoshinaga — APPROVED** [review]

File: README.rst [hunk 5]
```diff
@@ -141,7 +147,7 @@ To build the PHP driver for Windows:
    For :code:`<arch>`, :code:`<build>`, :code:`<visual studio version>`, and :code:`<path to PHP SDK>`, specify the same values
    that you used in the previous step.
 
-   For :code:`<full PHP version>`, specify the full version number of PHP that you want to install (e.g. :code:`8.1.18`).
+   For :code:`<full PHP version>`, specify the full version number of PHP binary you installed (e.g. :code:`8.1.18`).
```
[the remark this answers is not in the document]
Comment (sfc-gh-mheavin) [reply README.rst:150]: Done

**Pull request merged**